templates/v2/pro/list/_update_right_button.html.twig

```diff
@@ -0,0 +1,36 @@
+{% set currentUserRelay = relay ? user.userRelay(relay) : null %}
+{% if not currentUserRelay %}
+    {% set currentUserRelay = user.userRelays|length == 0 ? user.userRelays[0] : null %}
+{% endif %}
+
+{% set buttons = [
+    {
+        icon: 'user',
+        right: constant('App\\Entity\\MembreCentre::TYPEDROIT_GESTION_BENEFICIAIRES'),
+    },
+    {
+        icon: 'user-cog',
+        right: constant('App\\Entity\\MembreCentre::TYPEDROIT_GESTION_MEMBRES'),
+    },
+] %}
+
+{% for button in buttons %}
```

Comment:
This template could also be 

- A live component
- A turbo frame

It would solve all yout problems